### DISCUSSION TITLE
Multiple DNS Providers?

### DISCUSSION THREAD

**Discussion by sig9** [polymind-inc/acmebot#383] [General]
I am not sure if I encountered a feature or a Bug when I put in configuration data for two DNS providers at the same time.  When creating a cert I would only see one set of zones.  Testing the DNS configurations one at a time I see that they both work, just not concurrently for me.

**Comment by shibayan** [comment]
Multiple DNS providers are not supported. Only one will be used exclusively.

I would like to support this, but there are some issues that need to be addressed, so it will be addressed in the next major version.

**Reply by shibayan** [reply]
Issue created #386
